eng/Version.Details.xml

```diff
@@ -37,16 +37,16 @@
       Win-x64 is used here because we have picked an arbitrary runtime identifier to flow the version of the latest NETCore.App runtime.
       All Runtime.$rid packages should have the same version.
     -->
-    <Dependency Name="Microsoft.NETCore.App.Runtime.win-x64" Version="6.0.2" CoherentParentDependency="Microsoft.AspNetCore.Testing">
-      <Uri>https://dev.azure.com/dnceng/internal/_git/dotnet-runtime</Uri>
-      <Sha>839cdfb0ecca5e0be3dbccd926e7651ef50fdf10</Sha>
+    <Dependency Name="Microsoft.NETCore.App.Runtime.win-x64" Version="7.0.0-preview.2.22113.2" CoherentParentDependency="Microsoft.NET.Sdk.Razor">
+      <Uri>https://github.com/dotnet/runtime</Uri>
```

Comment:
Any issue with moving from internal azdo URI to github?

Reply:
No issue.